examples/complete/variables.tf

```diff
@@ -150,3 +150,9 @@ variable "apply_config_map_aws_auth" {
   default     = "false"
   description = "Whether to generate local files from `kubeconfig` and `config_map_aws_auth` and perform `kubectl apply` to apply the ConfigMap to allow the worker nodes to join the EKS cluster"
 }
+
```

Comment:
Didn't actually test what happens if you don't pass this variable and a blank string is used. This could error.